### PR TITLE
Feature/glossary update sept 2022

### PR DESCRIPTION
**Pull request by frivieremicroej**
this is a first draft for future glossary updates, please comment.

### CODE REVIEW

File: glossary.rst [hunk 1]
```diff
@@ -14,61 +14,58 @@ This glossary defines the technical terms upon which the `MICROEJ VEE (Virtual E
...
-            A Multi-Sandbox Firmware is a Firmware that implements the ability to be extended, by exposing a set of APIs and a memory space to link Sandboxed Applications. (formerly called a Multi-app Firmware) 
+         Multi-Sandbox Executable
+            A Multi-Sandbox Executable is an Executable that implements the ability to be extended, by exposing a set of APIs and a memory space to link Sandboxed Applications. (formerly called a Multi-app Firmware) 
 
    Foundation Library
       A Foundation Library is a library that provides core or hardware-dependent functionalities. A Foundation Library combines `managed code <https://en.wikipedia.org/wiki/Managed_code>`_ (Java) and low-level APIs (C) implemented by one or more Abstraction Layers through a native interface (:ref:`SNI <sni>`).
 
    Mock
-      A Mock is a mockup of a Board Support Package capability that mimics an hardware functionality for the Simulator.
+      A Mock is a mockup of a board support package (BSP) capability that mimics an hardware functionality for the Simulator.
```
Comment (jerome-leroux) [inline glossary.rst:50]:
```suggestion
      A Mock is a mockup of a board support package (BSP) capability that mimics a hardware functionality for the Simulator.
```

2 other occurrences identified in https://docs.microej.com/en/latest/overview/sdk.html?highlight=%22an%20hardware%22#microej-sdk and https://docs.microej.com/en/latest/PlatformDeveloperGuide/ecom-comm.html?highlight=%22an%20hardware%22#comm-port-identifier

**Review by jerome-leroux — CHANGES_REQUESTED** [review]

File: glossary.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@
 MicroEJ Glossary
 ================
 
-This glossary defines the technical terms upon which the `MICROEJ VEE (Virtual Execution Environment) <https://developer.microej.com/microej-vee-virtual-execution-environment>`_ is built.
+This glossary defines the technical terms upon which a `MICROEJ VEE (Virtual Execution Environment) <https://developer.microej.com/microej-vee-virtual-execution-environment>`_ is built.
```
Comment (frivieremicroej) [inline glossary.rst:7]: f/s/r decision: remove the line, keep VEE in the glossary.

```suggestion
```

File: glossary.rst [hunk 3]
```diff
@@ -14,61 +14,58 @@ This glossary defines the technical terms upon which the `MICROEJ VEE (Virtual E
       An Abstraction Layer is the C code that implements a Foundation Library's low-level APIs over a board support package (BSP) or a C library.
 
    Application
-      An Application is a software program that runs on a Powered by MicroEJ device.
+      An Application is a software program that runs on a MICROEJ VEE.
 
          Standalone Application
-            A Standalone Application is an Application that is directly linked to the C code to produce a Mono-Sandbox Firmware.
+            A Standalone Application is the main application that is booted by MICROEJ VEE. It is linked statically.
```
Comment (frivieremicroej) [inline glossary.rst:20]: f/s/r decision
```suggestion
            A Standalone Application is the main application that is booted by MICROEJ VEE. It is linked statically to produce a Mono-Sandbox Executable.
```

File: glossary.rst [hunk 4]
```diff
@@ -14,61 +14,58 @@ This glossary defines the technical terms upon which the `MICROEJ VEE (Virtual E
...
 
          Sandboxed Application
-            A Sandboxed Application is an Application that can run over a Multi-Sandbox Firmware. It can be linked either statically or dynamically.
+            A Sandboxed Application is an Application that can run over a Multi-Sandbox Executable. It can be linked either statically or dynamically.
```
Comment (frivieremicroej) [inline glossary.rst:23]: f/s/r decision
```suggestion
            A Sandboxed Application is an Application that can run over a Multi-Sandbox Executable. It is linked dynamically.
```

File: glossary.rst [hunk 5]
```diff
@@ -14,61 +14,58 @@ This glossary defines the technical terms upon which the `MICROEJ VEE (Virtual E
...
 
          System Application
-            A System Application (formerly called a Resident Application) is a Sandboxed Application that is statically linked to a Multi-Sandbox Firmware, as it is part of the initial image and cannot be removed.
+            A System Application (formerly called a Resident Application) is a Sandboxed Application that is statically linked to a Multi-Sandbox Executable, as it is part of the initial image and cannot be removed.
```
Comment (frivieremicroej) [inline glossary.rst:26]: f/s/r decision: remove System Application

File: glossary.rst [hunk 6]
```diff
@@ -14,61 +12,55 @@ This glossary defines the technical terms upon which the `MICROEJ VEE (Virtual E
...
       MicroEJ Module Manager (MMM) downloads, installs and controls the consistency of all the dependencies and versions required to build and publish a MicroEJ asset. It is based on `Semantic Versioning <https://semver.org>`_ specification.
 
-   Platform
-      A Platform integrates a MICROEJ VEE, an Architecture, one or more Foundation Libraries with their respective Abstraction Layers, and the board support package (BSP) for the target Device. It also includes associated Mocks for the Simulator.
+   VEE Port
+      A VEE Port is an implementation of MICROEJ VEE for a target device. It integrates an Architecture, one or more Foundation Libraries with their respective Abstraction Layers, and the board support package (BSP). It also includes associated Mocks for the Simulator. (formerly called Platform)
 
    SDK
-      MICROEJ SDK allows Firmware developers to build a Powered by MicroEJ device, by integrating an Architecture with both Java and C software on their device.
+      MICROEJ SDK is a comprehensive tools suite for developers to build VEE Ports for their devices, create Applications, build Executable and run Virtual Devices.
```
Comment (jerome-leroux) [inline glossary.rst:54]:
```suggestion
      MICROEJ SDK is a comprehensive tools suite for developers to build VEE Ports for their devices, create Applications, build Executable, and run Virtual Devices.
```

**Review by jerome-leroux — CHANGES_REQUESTED** [review]

File: glossary.rst [hunk 7]
```diff
@@ -14,61 +12,55 @@ This glossary defines the technical terms upon which the `MICROEJ VEE (Virtual E
       An Abstraction Layer is the C code that implements a Foundation Library's low-level APIs over a board support package (BSP) or a C library.
 
    Application
-      An Application is a software program that runs on a Powered by MicroEJ device.
+      An Application is a software program that runs on a MICROEJ VEE.
 
          Standalone Application
-            A Standalone Application is an Application that is directly linked to the C code to produce a Mono-Sandbox Firmware.
+            A Standalone Application is the main application that is booted by MICROEJ VEE. It is linked statically to produce a Mono-Sandbox Executable.
```
Comment (stephanemainchain) [inline glossary.rst:18]: booted: wording => never seen it used like this. Prefer launched?

File: glossary.rst [hunk 8]
```diff
@@ -14,61 +12,55 @@ This glossary defines the technical terms upon which the `MICROEJ VEE (Virtual E
...
-
-   VEE
-      MICROEJ VEE is an applications container. VEE stands for Virtual Execution Environment, and refers to the first implementation that embeds a virtual 32-bit processor, hence the term "Virtual". MICROEJ VEE runs on any OS/RTOS commonly used in embedded systems (FreeRTOS, QP/C, uc/OS, ThreadX, embOS, Mbed OS, Zephyr OS, VxWorks, PikeOS, Integrity, Linux, QNX, …) and can also run without RTOS (bare-metal) or proprietary RTOS. MICROEJ VEE includes the small MEJ32, along with a wide range of libraries (Add-On Libraries and Foundation Libraries).
+   VEE Port
+      A VEE Port is an implementation of MICROEJ VEE for a target device. It integrates an Architecture, one or more Foundation Libraries with their respective Abstraction Layers, and the board support package (BSP). It also includes associated Mocks for the Simulator. (formerly called Platform)
```
Comment (stephanemainchain) [inline glossary.rst:60]: for a target device or a target hardware ? (device could be reduced to mcu only?) just a suggestion

File: glossary.rst (hunk 6 above)
Comment (frivieremicroej) [reply glossary.rst]: done

File: glossary.rst (hunk 7 above)
Comment (frivieremicroej) [reply glossary.rst:18]: I replaced by executed

**Pull request closed without merging**